app/src/main/res/values/styles.xml

```diff
@@ -17,18 +17,21 @@
 <resources>
 
     <!-- Base application theme. -->
-    <style name="BaseAppTheme" parent="Theme.MaterialComponents.Light.DarkActionBar">
+    <style name="BaseAppTheme" parent="Theme.AppCompat.Light.NoActionBar">
```

Comment:
materialVersion = '1.1.0-alpha02' or newer

Reply:
Any reason?  Prefer to have stable version of libraries if possible

Reply:
Because here is a little strange that we use the different namespace of style or theme.
The `1.1.0-alpha02` or more, I think, it has now `Theme.MaterialComponents.NoActionBar` .